web/modules/custom/ecc_cct/ecc_cct.module

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use \Drupal\Core\Form\FormStateInterface;
+
+/**
+ * @file
+ * Primary module hooks for ECC Cookie Consent Tracker module.
+ */
+
+/**
+ * Implements hook_preprocess_html().
+ */
+function ecc_cct_preprocess_html(&$vars) {
+  if ($vars['root_path'] !== 'admin') {
```

Comment:
Changes as follows:

1. using eu cookie settings for excluded paths
2. using dependency injection for db service access
3. using config() from controllerbase to access config
4. using #config_target instead of default_value
5. js is printing nothing to console now
6. changed config schema to use integer as integer is what's actually used (boolean makes radio button choke)